app/views/tag/_tags_popover.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="main_div">
+  <p class="small_heading"> Follow related tags: </p>
+  <%= form_with remote: true, url: '/subscribe/multiple/tag/:tagnames' do |f| %>
+    <% tags.each do |tag| %>
+    <div>
+      <input type="checkbox" id="<%= tag.id %>" name="tagnames[]" value="<%= tag.name %>" <% if current_user.following(tag.name) %> checked <% end %> ><span class="label <%= label_name %>" style="margin-left: 3px; margin-right: 10px; display: inline-block;"> <%= tag.name %></span>
```

Comment:
Is the login modal linked after the tags are showed on the pop up on clicking subscribe or before showing them tags.
It should be linked before showing the tags else your code will show an error.

Reply:
If the user is not signed in and clicks on Subscribe+ button, login modal is shown instead of the popover.